spec/api/api.yaml

```diff
@@ -369,6 +389,17 @@ paths:
         For the snapped-shot documents that are not in the list of changes, the documents will be 
         verified as unchanged documents.
 
+        Verification Logic:
+        1. If the document specified in the delta doesn't exist in the snapshot, the 400 request error will be returned.
+        2. If the document specified in the delta with change type PURGE or DELETE, the corresponding snapshot document 
+           value in the snaphot MUST be null.
+        3. If the document specified in the delta with change type UPDATE, the expected document which is the 
+           corresponding snaphot document applied with the specified delta will be compared with the actual document 
+           which is the current document in the collection. The body of expected and actual document MUST be the same. 
+        4. When checking if the two blob objects are equals, the dictoinary properties of the two blobs MUST be the same.
+           In addition, the actual blob file in the database MUST exist; this can be checked by calling Database's 
```

Comment:
I guess this is ok, right?  There is no way for the file's contents to get out of sync with its name?

Reply:
With the recent CBSE bug that blob files in the database could be gone, I would like to check for the file existent as well. If this becomes extensive to test, we could have a separate API to test that.